### PR TITLE
Code Cleanup

### LINKED ISSUE #5: Use VBL logo for extension logo

### PR DESCRIPTION
**Pull request by braxtonhall**
No real fixes. Just made the code look a little bit prettier

edit: jk
resolves #5

### CODE REVIEW

File: extension/copy.js [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
+        const saveData = JSON.parse(localStorage.getItem(SAVE_DATA_KEY));
+        insertSaveData(parent, saveData);
+    };
 
 const getSaveData = (parent) => {
     const elements = getElementsByTags(parent, RELEVANT_TAGS);
-    return elements.reduce((acc, element) => {
-        if (element.id) {
+    return elements.reduce((saveData, element) => {
+        if (element.id && element.type !== 'hidden') {
```
Comment (adirar111) [inline extension/copy.js]: o.o so we're okay with not saving the hidden checkbox values?

File: extension/copy.js [hunk 2]
```diff
@@ -47,22 +51,24 @@ const insertSaveData = (parent, saveData) => {
...
+    appendButton(row, 'SAVE', saveData(editForm));
+    appendButton(row, 'LOAD', loadData(editForm));
+    const [body] = table.getElementsByTagName("tbody");
+    body.appendChild(row);
+};
 
-    });
+window.addEventListener("load", () => {
+    const [editForm] = document.getElementsByClassName("book_bit");
+    Array.from(document.getElementsByClassName("book_bitTable")).forEach(appendRow(editForm));
```
Comment (adirar111) [inline extension/copy.js]: epic

File: extension/ensureSaved.js [hunk 3]
```diff
@@ -1,66 +1,51 @@
...
+	});
 
 const addOnEditClicked = (element) =>
 	element.addEventListener("click", undoEdits);
 
 const observer = new MutationObserver((mutations) => {
-	addOnCancelEditButton();
+	/**
+	 * This potentially means things are added same listener to the same node multiple times,
+	 * but who cares if we set the edited boolean as true 8 times lmao
```
Comment (adirar111) [inline extension/ensureSaved.js:19]: AHAHAHAHA

File: extension/ensureSaved.js [hunk 4]
```diff
@@ -1,66 +1,51 @@
...
 
-const addOnCancelEditButton = () => [
+const addUndoEditListener = () => [
 	document.getElementById("book_editTabTextEditCancel1"),
 	document.getElementById("book_editTabTextEditCancel2"),
 	document.getElementById("book_editTabTextSave1"),
 	document.getElementById("book_editTabTextSave2"),
-	document.getElementById("book_editTabTextDelete"), // there's only one of these, so that it doesn't alert you when you're deleting something (?)
+	document.getElementById("book_editTabTextDelete"), // so that it doesn't alert you when you're deleting something (?)
 ].forEach(addOnEditClicked);
```
Comment (adirar111) [inline extension/ensureSaved.js:32]: this is a tad confuzzling -- we're trying to make it so that when these elements are clicked, we set `isEdited = false`, but `addOnEditClicked` doesn't really reflect that. might be a contender to rename in another PR (?)

File: extension/copy.js (hunk 1 above)
Comment (braxtonhall) [reply extension/copy.js]: Yeah I was looking through and I found some chaos in the system. LibraryThing relies on the frontend form input for the UUID of the form. So we'd be editing another entry if we don't do this

Comment (adirar111) [reply extension/copy.js]: oh yay, cool cool makes sense

File: extension/ensureSaved.js (hunk 4 above)
Comment (braxtonhall) [reply extension/ensureSaved.js:32]: OH oops yeah I forgot to update that name one sec

File: extension/ensureSaved.js [hunk 5]
```diff
@@ -29,7 +26,7 @@ const addUndoEditListener = () => [
 	document.getElementById("book_editTabTextSave1"),
 	document.getElementById("book_editTabTextSave2"),
 	document.getElementById("book_editTabTextDelete"), // so that it doesn't alert you when you're deleting something (?)
-].forEach(addOnEditClicked);
+].forEach((element) => element.addEventListener("click", undoEdits));
```
Comment (adirar111) [inline extension/ensureSaved.js:29]: 🙏

File: extension/copy.js [hunk 6]
```diff
@@ -18,6 +18,9 @@ const loadData = (parent) =>
 const getSaveData = (parent) => {
 	const elements = getElementsByTags(parent, RELEVANT_TAGS);
 	return elements.reduce((saveData, element) => {
+		// We can't change hidden elements because LibraryThing relies
+		// on hidden form inputs to send additional, form-specific metadata
+		// on save
```
Comment (adirar111) [inline extension/copy.js:23]: 🙌

**Review by adirar111 — APPROVED** [review]
W
